projects/github-actions/test-results-to-slack/tests/rules.test.js

```diff
@@ -0,0 +1,63 @@
+const crypto = require( 'crypto' );
```

Comment:
Fun fact: Github considered this file as "auto-generated", so it didn't expand it 🙃

Reply:
I noticed that too. I'm curious about what made GitHub do that.